db/factories/grant_factory.rb

```diff
@@ -1,5 +1,10 @@
 # frozen_string_literal: true
 
+def count_words(section_title, section_text)
+  str = "#{section_title} #{section_text}"
+  str.strip.split(/\s+/).length
+end
```

Comment:
i wonder if we should move this into a separate class so we can have a consistent way of counting section words on the backend. wyt?

maybe a `app/lib/word_counter.rb` class with this method in it?